src/objects.c

```diff
@@ -576,6 +576,8 @@ CK_RV p11prov_obj_from_handle(P11PROV_CTX *ctx, P11PROV_SESSION *session,
     CK_BBOOL token_supports_allowed_mechs = CK_TRUE;
     CK_RV ret;
 
+    P11PROV_debug("called");
+
```

Comment:
not sure we really need this

Reply:
without these logs, I was unable to track what is going on in the provider as there were just very rough. For this case, I could see that the PKCS#11 functions got called, but I did not see where from, if they came from the right context and so on. I do not mind downgrading the log level though.

Reply:
dropped